schema/contract-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$id": "http://example.com/example.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "default": {},
+  "description": "The root schema comprises the entire JSON document for contract configuration schema.",
+  "required": [
+      "title",
+      "networks",
+      "functions"
+  ],
+  "title": "Vesting contract for DXD token",
+  "networks": {
+      "mainnet": "0x0b17cf48420400e1D71F8231d4a8e43B3566BB5B", 
+      "rinkeby": "0x0b17cf48420400e1D71F8231d4a8e43B3566BB5B", 
+      "gnosis": "0x0b17cf48420400e1D71F8231d4a8e43B3566BB5B", 
+      "arbitrum": "0x0b17cf48420400e1D71F8231d4a8e43B3566BB5B", 
+      "arbitrumRinkeby": "0x0b17cf48420400e1D71F8231d4a8e43B3566BB5B"
+  },
+  "functions": [
+      {
+          "title": "Create vesting contract",
+          "functionName": "create(address, uint256, uint256, uint256, uint256)",
```

Comment:
Maybe this could be just the function name (just `create` instead of `create(address, uint256, uint256, uint256, uint256)`. We can always get the param types from the params array. Otherwise it becomes complicated to format it the way we want:

![image](https://user-images.githubusercontent.com/25136207/156999921-7872a9d8-6e31-4bed-9541-9d1465706043.png)

Reply:
Good idea!